<?xml version="1.0"?>

<launch>
    <node pkg="ndt_localizer" type="map_matcher" name="map_matcher">
        <remap from="scan_cloud" to="velodyne_points"/>
        <remap from="map_cloud" to="cloud_pcd"/>
        <param name="leaf_size" value="0.5"/>
        <param name="epsilon" value="1e-2"/>
        <param name="max_iteration" value="30"/>
        <param name="range" value="50"/>
    </node>
    <node pkg="ndt_localizer" type="ndt_odom_integrator" name="ndt_odom_integrator">
        <remap from="odom" to="odom"/>
        <remap from="imu/data" to="imu/data"/>
        <remap from="map_cloud" to="cloud_pcd"/>
        <param name="sigma_odom" value="0.01"/> -->
        <param name="sigma_imu" value="0.01"/>
        <param name="sigma_ndt" value="0.1"/>
    </node>
</launch>
